src/applications/vre/28-1900/wizard/pages/veteran/02-yesDisabilityRating.jsx

```diff
@@ -31,6 +32,9 @@ const YesDisabilityRating = props => {
       <Link
         aria-describedby="orientation-complete-notification"
         onClick={() => {
+          recordEvent({
+            event: 'ch31-orientation-skip',
+          });
```

Comment:
Sorry! Not sure how I missed this PR. Apologies for letting it sit out in cyberspace. In addition to updating the event name to `howToWizard-skip-orientation`, it also needs to be added [here](https://github.com/department-of-veterans-affairs/vets-website/blob/7fc4f2d72dd237f253f213f672583e1ecc895e19/src/applications/vre/28-1900/wizard/pages/service-member/02-yesVaMemorandum.jsx#L31-L40) and [here](https://github.com/department-of-veterans-affairs/vets-website/blob/7fc4f2d72dd237f253f213f672583e1ecc895e19/src/applications/vre/28-1900/wizard/pages/service-member/03-yesIDES.jsx#L31-L40)

Reply:
@micahchiang Yeah I had another commit that I needed to push, now both the event name has been updated and has been added to the two other places called out